src/commands/appdistribution-testers-add.ts

```diff
@@ -0,0 +1,28 @@
+import {Command} from "../command";
+import * as utils from "../utils";
+import {requireAuth} from "../requireAuth";
+import {FirebaseError} from "../error";
+import {needProjectNumber} from '../projectUtils';
+import {AppDistributionClient} from '../appdistribution/client';
+import {getEmails} from '../testerEmailParser';
+
+module.exports = new Command("appdistribution:testers:add [emails...]")
+    .description("Create Testers")
+    .option(
+        "--file <file>",
+        "a path to a file containing a comma separated list of tester emails to be added"
```

Comment:
It's a little weird that we specify comma separated list. It can be comma separated or it can be newline separated. Also weird because the `emails...` property isn't a comma separated list, it's a space separated list.

Basically maybe reword it to be `a path to a file containing a list of tester emails to be added`